src/components/header-card/header-card.tsx

```diff
@@ -1,18 +1,19 @@
 import React, { FC } from 'react';
-import { View } from 'react-native';
+import { View, ViewStyle } from 'react-native';
 
 import { InsetSubstitute } from '../inset-substitute/inset-substitute';
 import { useHeaderCardStyles } from './header-card.styles';
 
 interface Props {
   hasInsetTop?: boolean;
+  style?: ViewStyle;
 }
 
-export const HeaderCard: FC<Props> = ({ hasInsetTop = false, children }) => {
+export const HeaderCard: FC<Props> = ({ hasInsetTop = false, children, style }) => {
   const styles = useHeaderCardStyles();
 
   return (
-    <View style={styles.container}>
+    <View style={[styles.container, style]}>
```

Comment:
`styles.container` has `generateShadow(23, colors.black10)` why it does not work?

Reply:
I think HeaderCard should have only 1 version of shadow

Reply:
Sorry, I don't know about iOS but, as for me, it's not displayed on Android because of color transparency

Reply:
Sorry, almost that... You need a color picker to notice a slight difference of colors

Reply:
yeah, do that and we will test that